app/assets/javascripts/parent-taxon-prefix-preview.js

```diff
@@ -5,31 +5,43 @@
     this.start = function(el) {
       var $parentSelectEl = $(el).find('select.js-parent-taxon');
       var $pathPrefixEl = $(el).find('.js-path-prefix-hint');
+      var $taxonBasePathEl = $(el).find('#taxon_base_path');
 
-      updateBasePathPreview();
-      $parentSelectEl.change(updateBasePathPreview);
-
-      function updateBasePathPreview() {
+      function getParentPathPrefix(callback) {
         var parentTaxonContentId = $parentSelectEl.val();
 
         if (parentTaxonContentId.length === 0) {
-          $pathPrefixEl.addClass('hidden');
-          $pathPrefixEl.text('');
+          callback();
           return;
         }
 
         $.getJSON(
           window.location.origin + '/taxons/' + parentTaxonContentId + '.json'
         ).done(function(taxon) {
-          if (typeof taxon.path_prefix !== 'undefined') {
-            $pathPrefixEl.html('Base path must start with <b>/' + taxon.path_prefix + '</b>');
+          callback(taxon.path_prefix);
+        });
+      }
+
+      function updateBasePathPreview() {
+        getParentPathPrefix(function (path_prefix) {
+          if (path_prefix) {
+            $pathPrefixEl.html('Base path must start with <b>/' + path_prefix + '</b>');
             $pathPrefixEl.removeClass('hidden');
           } else {
             $pathPrefixEl.addClass('hidden');
             $pathPrefixEl.text('');
           }
         });
       }
+
+      getParentPathPrefix(function (path_prefix) {
+        if (path_prefix) {
+          $taxonBasePathEl.val('/'+ path_prefix + '/');
```

Comment:
This value change seems to have an undesired effect on integration: for a valid base path, the base path field is being modified from `/education/school-places` to `/education/` on https://content-tagger.integration.publishing.service.gov.uk/taxons/4dafe1f8-593c-47da-a3b5-577d96feeee4/edit

Reply:
Yep, I was a little too enthusiastic, I've got a fix up here https://github.com/alphagov/content-tagger/pull/687